website_product_configurator/static/src/js/config_form.esm.js

```diff
@@ -0,0 +1,629 @@
+/** @odoo-module **/
+
+import Dialog from "@web/legacy/js/core/dialog";
+import {insertThousandsSep} from "@web/core/utils/numbers";
+import {jsonrpc} from "@web/core/network/rpc_service";
+import {localization} from "@web/core/l10n/localization";
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.ProductConfigurator = publicWidget.Widget.extend({
+    selector: ".product_configurator",
+    events: {
+        "change.datetimepicker #product_config_form .input-group.date":
+            "_onChangeDateTime",
+        "change #product_config_form .config_attribute": "_onChangeConfigAttribute",
+        "change #product_config_form .custom_config_value.config_attachment":
+            "_onChangeFile",
+        "change #product_config_form .custom_config_value": "_onChangeCustomField",
+        "click #product_config_form .config_step": "_onClickConfigStep",
+        "click #product_config_form .btnNextStep": "_onClickBtnNext",
+        "click #product_config_form .btnPreviousStep": "_onClickBtnPrevious",
+        "submit #product_config_form": "_onSubmitConfigForm",
+        "click #product_config_form .image_config_attr_value_radio":
+            "_onClickRadioImage",
+        "change #product_config_form .custom_config_value.spinner_qty":
+            "_onChangeQtySpinner",
+        "click #product_config_form .js_add_qty": "_onClickAddQty",
+        "click #product_config_form .js_remove_qty": "_onClickRemoveQty",
+    },
+
+    init: function () {
+        this._super.apply(this, arguments);
+        this.config_form = $("#product_config_form");
```

Comment:
IMO this should be in `start` rather than `init`

And ideally we should use either vanilla-js or `this.$`